File: MyWebView/app/src/main/java/com/hiro/mywebview/MainActivity.java
package com.hiro.mywebview;

import android.os.Bundle;
import android.app.Activity;
import android.webkit.WebView;
import android.webkit.WebViewClient;
import android.webkit.WebSettings;
import com.google.android.material.floatingactionbutton.FloatingActionButton;
import com.google.android.material.snackbar.Snackbar;
import android.view.View;
import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.Toolbar;
public class MainActivity extends AppCompatActivity {

    private static final int REQUEST_CODE = 1;

       private WebView webView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

	

        Toolbar toolbar = findViewById(R.id.tool_bar);
        setSupportActionBar(toolbar);
    }
    
}

File: MyApplication/app/src/main/java/com/hiro/take/myapplication/MainActivity.kt
package com.hiro.take.myapplication
import android.os.Bundle
import android.webkit.WebView
import android.webkit.WebViewClient
import android.app.Activity

class MainActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.main)
        val myWebView: WebView = findViewById(R.id.webview)
					 myWebView.loadUrl("https://www.google.co.jp/")
			
		}
		
}
